Add note about BigInt polyfills to README (#496)
xmtp/xmtp-js: README.md

## Troubleshoot

### Buffer polyfill

> If you get into issues with Buffer and polyfills check out our [solutions](https://xmtp.org/docs/faq#why-my-app-is-failing-saying-buffer-is-not-found).

Reviewer: ```suggestion
If you run into issues with Buffer and polyfills, see this [solution](https://xmtp.org/docs/faq#why-my-app-is-failing-saying-buffer-is-not-found).
```

Author: BTW, we may want to update this sentence on our docs site to match. first sentence in the linked section: https://xmtp.org/docs/faq#why-my-app-is-failing-saying-buffer-is-not-found.

### BigInt polyfill

> Some older browsers do not support `BigInt` (see [compatibility list](https://caniuse.com/bigint)). Unfortunately, the way this SDK uses `BigInt` is incompatible with polyfills. To ensure that a polyfill is not added to your application bundle, update your [browserslist](https://github.com/browserslist/browserslist) configuration to exclude browsers that don't support `BigInt`.

Reviewer: ```suggestion
This SDK uses `BigInt` in a way that's incompatible with polyfills. To ensure that a polyfill isn't added to your application bundle, update your [browserslist](https://github.com/browserslist/browserslist) configuration to exclude browsers that don't support `BigInt`.

For the list of browsers that don't support `BigInt`, see this [compatibility list](https://caniuse.com/bigint).
```
